src/js/actions/transform.js

```diff
@@ -1154,13 +1192,7 @@ define(function (require, exports) {
                 .unshift(documentRef)
                 .toArray(),
             rotateObj = layerLib.rotate(layerRef, angle),
-            options = {
-                historyStateInfo: {
-                    name: nls.localize("strings.ACTIONS.ROTATE_LAYERS"),
-                    target: documentLib.referenceBy.id(document.id)
-                }
-            },
-            historyPromise = this.transfer(historyActions.newHistoryState, document.id,
+            historyPromise = coalesce ? Promise.resolve() : this.transfer(historyActions.newHistoryState, document.id,
```

Comment:
cool.  I had considered an alternate approach of providing a coalesce parameter to `newHistoryState` which would then pass that along to the event emitted to the history store... but I see no functional advantage to that approach, nor much of a "readability" advantage.